npm-package/src/controllers/otelController.test.js

```diff
@@ -0,0 +1,230 @@
+import { otelController } from './otelController'
```

Comment:
Previously, we had test files in the tests folder. I noticed that your PR has now added test files to be in the same directory as the files being test. I would like to get everyone's thoughts on this, but I believe test files should be in their own separate folder for these reasons:

1. For organization purposes, I believe it is more clear to have actual source code separate from the test files. Extra test files for each source code file clutters the source code directories. It is for the same reason that we have a separate dist folder to hold the .js files for each corresponding .ts file. I think having the test file name as it is currently is sufficient to know what source file the test file tests.

2. For easier version control, having test files in a separate folder would prevent them from accidentally being included in builds. This prevents potential issues of shipping test files to production.

Again, I would like to get second opinions.